src/pages/_proxied-dot-io/vagrant/downloads/index.jsx

```diff
@@ -46,12 +46,12 @@ function DownloadsPage({ product, releases, latestVersion }) {
 			packageManagerOverrides={[
 				{
 					label: 'Homebrew',
-					commands: [`brew install vagrant`],
+					commands: [`brew install hashicorp/tap/hashicorp-vagrant`],
 					os: 'darwin',
 				},
 				{
 					label: 'Homebrew',
-					commands: [`brew install vagrant`],
+					commands: [`brew install hashicorp/tap/hashicorp-vagrant`],
```

Comment:
oh actually, I think we need to revert this since only darwin AMD is supported

Reply:
@BRKalow are we ok to remove this whole section? https://github.com/hashicorp/dev-portal/blob/64af4af39b67eb9763cefaafa1486c57d71a5ab2/src/pages/_proxied-dot-io/vagrant/downloads/index.jsx#L52-L56 This will not work for users

Reply:
I went ahead and removed this 👍🏻